helix-core/src/textobject.rs

```diff
@@ -260,18 +260,47 @@ pub fn textobject_treesitter(
     object_name: &str,
     slice_tree: Node,
     lang_config: &LanguageConfiguration,
+    better_capture: bool,
```

Comment:
I think this should more closely express what's going on. Why would you ever not use the `better_capture`? Maybe even use an enum here, to make the call site more explicit/readable?

Reply:
Yeah that's Temporary name, I thought someone reviewing the PR could suggest a Better name